lib/mongoid/contextual/mongo.rb

```diff
@@ -680,6 +694,107 @@ def _session
       def acknowledged_write?
         collection.write_concern.nil? || collection.write_concern.acknowledged?
       end
+
+      # Extracts the value for the given field name from the given attribute
+      # hash.
+      #
+      # @param [ Hash ] attrs The attributes hash.
+      # @param [ String ] field_name The name of the field to extract.
+      #
+      # @param [ Any ] The value for the given field name
```

Comment:
I usually use `Object` to indicate any type of value is acceptable.

Reply:
done